src/NodeVisitor/ParentClassNameCollectingNodeVisitor.php

```diff
@@ -40,10 +40,19 @@ public function getParentClassNames(): array
         sort($uniqueParentClassNames);
 
         // remove native classes
-        $namespacedClassNames = array_filter(
-            $uniqueParentClassNames,
-            fn (string $parentClassName): bool => str_contains($parentClassName, '\\')
```

Comment:
btw: I don't like these short function stuff - especially with `array_filter` since its hard to change and hard to debug. as soo as I want to change the line - e.g. for debuggin - I need to restructure the code because the short lambda no longer works.

just personal preference though

Reply:
I prefer it because it forces write to make condition simple and basic :+1:  Anything more complex is usually harder to read and doing too many filterings at once.